hpat/tests/test_series.py

```diff
@@ -1265,6 +1265,14 @@ def test_impl(S, values):
         values = {'b', 'c', 'e'}
         pd.testing.assert_series_equal(hpat_func(S, values), test_impl(S, values))
 
+    def test_series_isna(self):
```

Comment:
We already have such test in test_series: test_series_isna1. Please, use it or specify name of these tests like: test_series_isnan_int and test_series_isnan_float